event-checklist.md

```diff
@@ -14,12 +14,16 @@ The steps can be summarised as;
 8. Ensure Post Event Report is written and distributed
 
 ##Checklist for Event Teams
+
 * [ ] Get your event team together and ensure you have a balanced mix of skills
 * [ ] Read through the [event subcommittee policy](subcommittee_policy_v2.md)
 * [ ] Ensure all members of your team are [members of Linux Australia](https://www.linux.org.au/membership)
-* [ ] Raise a motion to form a new subcommittee by [emailing the Council](mailto:council@linux.org.au). A draft motion is provided below.
-* [ ] Understand expectations from Linux Australia regarding the percentage of event profits that are expected to be returned to Linux Australia
+* [ ] Request a new subcommittee be created by [emailing the Council](mailto:council@linux.org.au). An example of what should be included in the request is given below.
+* [ ] Understand expectations from Linux Australia that the event is expected to make a profit of $AUD 11 per attendee, or as negotiated, and that all profits from the event are returned to Linux Australia to be re-invested in the region's Linux and open source community.

```

Comment:
So the $11 per attendee isn't really considered "profit" from an event. That amount is very roughly the admin cost to LA for the event. So it should appear as an expense in the events P&L. Thus the profit an event returns should be /after/ the $11 per attendee.

Some events have called this the LA tax which is probably apt.

There is wording in the subcommittee policy for profit where we may ask an event to aim for $X return which is in addition to this expense accounting.